### PR TITLE
Feature/verifier domain audit, #18

### PR DESCRIPTION
**Pull request by bestwebua**
- [x] Added PTR record audit, https://github.com/rubygarage/truemail/issues/18
- [x] Updated documentation
- [x] Updated wiki
- [x] Updated gem version

### CODE REVIEW

File: lib/truemail/audit/ptr.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+      def current_host_reverse_lookup
+        IPAddr.new(current_host_address).reverse
+      end
+
+      def ptr_records
+        @ptr_records ||= Truemail::Wrapper.call do
+          Resolv::DNS.new.getresources(
+            current_host_reverse_lookup, Resolv::DNS::Resource::IN::PTR
+          ).map { |ptr_record| ptr_record.name.to_s }
```
Comment (alexxty7) [inline lib/truemail/audit/ptr.rb]: { |ptr_record| ptr_record.name.to_s } - extract method

File: lib/truemail.rb [hunk 2]
```diff
@@ -29,12 +26,23 @@ def reset_configuration!
...
       Truemail::Validator.new(email, **options).run
     end
 
     def valid?(email, **options)
       validate(email, **options).result.valid?
     end
+
+    def host_audit
+      raise_unless(configuration, NOT_CONFIGURED)
+      Truemail::Auditor.new
```
Comment (alexxty7) [inline lib/truemail.rb]: maybe will be better to have run class method. - Truemail::Auditor.run

Comment (bestwebua) [reply lib/truemail.rb]: Yup!

File: lib/truemail/audit/ptr.rb (hunk 1 above)
Comment (bestwebua) [reply lib/truemail/audit/ptr.rb]: Leaved as is. It's have used above in run method too.

**Pull request merged**